README.md

```diff
@@ -7,667 +7,73 @@
-             "gamma": gamma,
-             "objective": "multi:softprob"}
-
-    xgb_model = xgb.train(param, dtrain, steps)
-
-    preds = xgb_model.predict(dtest)
-    best_preds = np.asarray([np.argmax(line) for line in preds])
-
-    context.log_result('accuracy', float(accuracy_score(Y_test, best_preds)))
-    context.log_model('model', body=bytes(xgb_model.save_raw()), 
-                      model_file='model.txt', 
-                      metrics=context.results, parameters={'xx':'abc'},
-                      labels={'framework': 'xgboost'},
-                      artifact_path=context.artifact_subpath('models'))
-```
-
-The example training function can be executed locally with parameters, and the run results and artifacts can be logged automatically into a database by using a single command, as demonstrated in the following example; the example sets the function's `eta` parameter:
-```python
-train_run = run_local(handler=xgb_train, pramas={'eta': 0.3})
-```
-
-Alternatively, you can replace the function with a serverless runtime to run the same code on a remote cluster, which could result in a ~10x performance boost.
-You can find examples for different runtimes &mdash; such as a Kubernetes job, Nuclio, Dask, Spark, or an MPI job &mdash; in the MLRun [**examples**](examples) directory.
-
-If you run your code from the `main` function, you can get the runtime context by calling the `get_or_create_ctx` method, as demonstrated in the following code from the MLRun [**training.py**](examples/training.py) example application.
-The code also demonstrates how you can use the context object to read and write execution metadata, parameters, secrets, inputs, and outputs:
-
-```python
-from mlrun import get_or_create_ctx
-from mlrun.artifacts import ChartArtifact
-import pandas as pd
-
-
-def my_job(context, p1=1, p2='x'):
-    # load MLRUN runtime context (will be set by the runtime framework e.g. KubeFlow)
-
-    # get parameters from the runtime context (or use defaults)
+MLRun uses Serverless Function technology: write the code once, using your preferred development environment and simple “local” semantics, and then run it as-is on different platforms and at scale. MLRun automates the build process, execution, data movement, scaling, versioning, parameterization, outputs tracking, CI/CD integration, deployment to production, monitoring, and more.
 
-    # access input metadata, values, files, and secrets (passwords)
-    print(f'Run: {context.name} (uid={context.uid})')
-    print(f'Params: p1={p1}, p2={p2}')
-    print('accesskey = {}'.format(context.get_secret('ACCESS_KEY')))
-    print('file\n{}\n'.format(context.get_input('infile.txt', 'infile.txt').get()))
-    
-    # Run some useful code e.g. ML training, data prep, etc.
+Those easily developed data or ML “functions” can then be published or loaded from a marketplace and used later to form offline or real-time production pipelines with minimal engineering efforts.
```

Comment:
loaded from a [marketplace]..  add hyperlink to mlrun marketplace (https://www.mlrun.org/marketplace/)